python_transport/wirepas_gateway/protocol/mqtt_wrapper.py

```diff
@@ -210,26 +227,47 @@ def run(self):
             # thread has exited
             self.on_termination_cb()
 
+    def _publish_from_wrapper_thread(self, topic, payload, qos, retain):
+        """
+        Internal method to publish on Mqtt. This method is only called from
+        mqtt wrapper thread to avoid races.
+        :param topic: Topic to publish on
```

Comment:
Please use [this style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for the doc strings since it is the one we _try_ to observe